### DISCUSSION TITLE
Deploying from a different branch than main.

### DISCUSSION THREAD

**Discussion by Mischa-bc** [vercel/community#3394] [Help]
### Summary

I'm trying to deploy another project from a different branch than main.

I went into "Settings" and "Git" in my project and changed name of the branch in the "Production Branch" field from "main" to a different existing branch. Clicked "save".

But nothing has changed, it's still deploying from the main branch. I've tried redeploying, still nothing has changed.

On the main page it still says "source: main"

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/24721856/0437fc16-cda1-4621-a586-7472a76160bd)

![image](https://github.com/vercel/community/assets/24721856/2fe9baa4-6126-4172-8761-336bb0d03ce4)

**Comment by amyegan** [comment]
Hey @Mischa-bc. If you push a new commit to the `light` branch, it should trigger a new production deployment from that branch. Does that work for you?
